jest.integration.config.ts

```diff
@@ -4,7 +4,7 @@ const config: Config.InitialOptions = {
   coveragePathIgnorePatterns: ['/node_modules/', '/dist/'],
   testPathIgnorePatterns: ['/src/'],
   preset: 'ts-jest',
-  setupFiles: ['<rootDir>/.integration.test.env'],
+  setupFiles: ['<rootDir>/.integration.test-dev.env'],
```

Comment:
How does this work dynamically if we hardcode to one environment here?

Reply:
I'm assuming you're overriding in the scripts in package.json

Reply:
Yes I am.